### PR TITLE
Show recipients in envelopes' list when displaying a 'sent' folder

### PR DESCRIPTION
**Pull request by StCyr**
Fix to https://github.com/nextcloud/mail/issues/2019

image

### CODE REVIEW

File: src/components/Envelope.vue [hunk 1]
```diff
@@ -96,14 +96,26 @@ export default {
...
+		addresses() {
+			if (this.folder.specialRole === 'sent') {
+
+				let recipients = [this.data.to, this.data.cc].flat().map(function(recipient) {
+					return recipient.label ? recipient.label : recipient.email
+				})
+
+				return recipients.length > 0 ? recipients.join(', ') : t('mail', 'Bcc recipients only')
+
+			} else {
```
Comment (ChristophWurst) [inline src/components/Envelope.vue]: no `else` needed as the `if` will return early

**Comment by ChristophWurst** [comment]
Btw, I think you should check your git config. Github does not associate the commit with your user. The email addresses might not match.

File: src/components/Envelope.vue (hunk 1 above)
Comment (StCyr) [reply src/components/Envelope.vue]: sorry, I don't get you: `else` will be executed when not in a 'sent folder' (or at least it's what I'm expecting :-) )

Should I amend all my commits to set the proper email address?

Comment (ChristophWurst) [reply src/components/Envelope.vue]: it's about simpler flow control:

this
```js
if (something) {
    ...
    return
} else {
   ...
}
```
versus

```js
if (something) {
    ...
    return
}

...
```

-> it's the same thing ;)

> Should I amend all my commits to set the proper email address?

Yes please :)

**Comment by StCyr** [comment]
> 
> 
> Btw, I think you should check your git config. Github does not associate the commit with your user. The email addresses might not match.

I've added this email address to my github's config.

**Comment by ChristophWurst** [comment]
One more lint autofix then we can merge :)

**Comment by StCyr** [comment]
should be ok now

**Pull request merged**